server/src/main/scala/test/TestDrtSystem.scala

```diff
@@ -206,9 +224,9 @@ case class TestDrtSystem(airportConfig: AirportConfig, params: DrtParameters)
       val startDay = SDate.now()
       DateRange.utcDateRange(startDay, startDay.addDays(30)).map(day => {
         val arrivals = CSVFixtures.csvPathToArrivalsOnDate(day.toISOString, file)
-          .collect {
-            case Success(arrival) => arrival
-          }
+                                  .collect {
```

Comment:
I'm not sure if intellij is indenting these new lines like this but we don't want to change from how we've always indented, ie with the new line 2 spaces in from the line above. It's only the indentation for arguments in a function or class definition that are indented this way